ingredients/file/fileContent.go

```diff
@@ -101,18 +101,16 @@ func (f File) content(ctx context.Context, test bool) (types.Result, error) {
 			if err != nil {
 				return types.Result{
 					Succeeded: false, Failed: true,
-					Changed: false, Notes: []fmt.Stringer{
```

Comment:
Can we get a note here for the parse error? The error itself doesn't have the generated filename, for example, which could be helpful